configuration/config-txt/boot.md

```diff
@@ -113,7 +113,11 @@ On the Raspberry Pi 4B, if this value is set to `0` then the interrupts will be
 
 ## force_eeprom_read
 
-Set this option to `0` to prevent the firmware from trying to read an I2C HAT EEPROM (connected to pins ID_SD & ID_SC) at powerup.
+Set this option to `0` to prevent the firmware from trying to read an I2C HAT EEPROM (connected to pins ID_SD & ID_SC) at powerup. See also `disable_poe_fan`.
+
+## disable_poe_fan
+
+Set this option to `1` to prevent control of the PoE HAT fan through I²C on pions ID_SD & ID_SC). Without this, a probe on the I²C bus will happen at startup, even when not using the PoE HAT.
```

Comment:
pions -> pins ? :wink:

Reply:
@JamesH65 Should this use `I2C` (instead of `I²C`) to match the previous paragraph?

Reply:
Fixed the typo, and I also changed to `I2C` (typing `I²C` is just a habit, but everywhere else uses I2C, so better be consistent). I forced pushed the fixes.